src/test/java/unit/AuthLoginLogout.java

```diff
@@ -64,14 +58,35 @@ void loginTestUser() throws IOException {
         equalToIgnoringCase(currentTerraUser.get().terraUserEmail));
   }
 
+  @Test
+  @DisplayName("test user logout updates global context")
+  void logoutTestUser() throws IOException {
+    // select a test user and login
+    TestUsers testUser = TestUsers.chooseTestUser();
+    testUser.login();
+
+    // `terra auth revoke`
+    TestCommand.Result cmd = TestCommand.runCommand("auth", "revoke");
+    assertEquals(0, cmd.exitCode);
+
+    // check that the credential store on disk is empty
+    DataStore<StoredCredential> dataStore = TestUsers.getCredentialStore();
+    assertEquals(0, dataStore.keySet().size(), "credential store is empty");
+
+    // read the global context in from disk again to check what got persisted
+    // check that the current user in the global context is unset
+    GlobalContext globalContext = GlobalContext.readFromFile();
+    Optional<TerraUser> currentTerraUser = globalContext.getCurrentTerraUser();
+    assertFalse(currentTerraUser.isPresent(), "current user unset in global context");
```

Comment:
you could also assert `isEmpty()`

Reply:
those seem equivalent in readability to me.